### PR TITLE
Update HardwareSettingsController View to include Save/Load Buttons

### PR DESCRIPTION
**Pull request by retiutut**
- [x] Start using Cp5 buttons
- [x] Make a strong effort to use existing styles
- [x] Leave room for optional "Send Settings" button in the future

### CODE REVIEW

File: OpenBCI_GUI/Interactivity.pde [hunk 1]
```diff
@@ -294,6 +294,36 @@ synchronized void mouseReleased() {
 //------------------------------------------------------------------------
 //                       Classes
 //------------------------------------------------------------------------
+/*
```
Comment (Andrey1994) [inline OpenBCI_GUI/Interactivity.pde:297]: lets remove it

**Review by Andrey1994 — APPROVED** [review]

File: OpenBCI_GUI/Interactivity.pde (hunk 1 above)
Comment (retiutut) [reply OpenBCI_GUI/Interactivity.pde]: Done!

**Review by Andrey1994 — APPROVED** [review]

File: OpenBCI_GUI/ADS1299SettingsController.pde [hunk 2]
```diff
@@ -257,4 +283,79 @@ class ADS1299SettingsController {
...
+                        selectOutput("Save settings to file", "storeHardwareSettings");
+                    }
+                });
+                break;
+            default:
+                break;
+        }
+        return myButton;
+    }
 };
```
Comment (daniellasry) [inline OpenBCI_GUI/ADS1299SettingsController.pde:333]: Suggestion for the future, avoid the switch statement. Have a createHWSettingsLoadButton() and createHWSettingsSaveButton(). Have both these functions call createButton() and then add the appropriate callback.

File: OpenBCI_GUI/W_TimeSeries.pde [hunk 3]
```diff
@@ -183,43 +186,14 @@ class W_timeSeries extends Widget {
...
-            hardwareSettingsStoreButton.setColorNotPressed(color(57,128,204));
-            hardwareSettingsStoreButton.textColorNotActive = color(255);
-            hardwareSettingsStoreButton.hasStroke(false);
-            hardwareSettingsStoreButton.setHelpText("Save current settings to file.");
-        }
-
-        int x_hsc = int(ts_x);
-        int y_hsc = int(ts_y);
-        int w_hsc = int(ts_w); //width of montage controls (on left of montage)
-        int h_hsc = int(ts_h); //height of montage controls (on left of montage)
```
Comment (daniellasry) [inline OpenBCI_GUI/W_TimeSeries.pde:219]: nice, much cleaner now

File: OpenBCI_GUI/ADS1299SettingsController.pde (hunk 2 above)
Comment (retiutut) [reply OpenBCI_GUI/ADS1299SettingsController.pde:333]: Sweet. Will do.

**Pull request merged**